lib/kafka-consumer.js

```diff
@@ -45,39 +44,39 @@ Consumer.prototype.getConsumer = Promise.method(function (opts, topts) {
   });
 
   // hack node-rdkafka
-  consumer.__kafkaAvro_getReadStream = consumer.getReadStream;
-  consumer.getReadStream = this._getReadStreamWrapper.bind(this, consumer);
 
   consumer.__kafkaAvro_on = consumer.on;
   consumer.on = this._onWrapper.bind(this, consumer);
 
   return consumer;
 });
 
-/**
- * The node-rdkafka getReadStream method wrapper, will deserialize
- * the incoming message using the existing schemas.
- *
- * @param {kafka.KafkaConsumer} consumerInstance node-rdkafka instance.
- * @param {string} topic Topic to consume on.
- * @param {Object=} opts Stream options.
- * @return {Stream} A Stream.
- */
-Consumer.prototype._getReadStreamWrapper = function (consumerInstance, topic,
-  opts) {
+Consumer.prototype.getConsumerStream = Promise.method(function (opts, topts) {
```

Comment:
Can you explain what happened here? I see a private method get converted into public

Reply:
it's not replacing the old private one, it's a new public. 

I will provide the JSDOC here - soon I commit this